rc/tools/go/gopls.kak

```diff
@@ -0,0 +1,98 @@
+# gopls.kak: gopls bindings for kakoune
+
+define-command -docstring "Enter gopls mode.
```

Comment:
You could make this docstring more readable by using the relatively unused format:

https://github.com/mawww/kakoune/blob/ec3d7c31040afaee65afde15832d9b0ed6b2224d/rc/tools/lint.kak#L2-L12

Reply:
Thanks, I will.